psst-gui/src/main.rs

```diff
@@ -54,7 +59,12 @@ fn main() {
         WebApi::global().load_local_tracks(state.config.username().unwrap());
     } else {
         // No configured credentials, open the account setup.
-        let window = ui::account_setup_window();
+        let mut window = ui::account_setup_window();
+
+        if state.config.kiosk_mode {
+            window = ui::kiosk_setup_window();
+        }
```

Comment:
Should we conditionally define the window rather than making it mutable and then switching?

Reply:
I think this is alright as the most likely one to happen would be the account set up rather than kiosk set up